caput/fft/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""caput.fft.
+
+Utilities for FFTs.
+
+Features
+========
+- Multi-threaded `scipy.fft` by default
+- Helpers to use `pyFFTW`
+"""
+
+from ._scipy_fft import *  # noqa: F403
+from . import _fftw as fftw
```

Comment:
If we're doing this, is there any reason to call the file `_fftw.py` instead of just `fftw.py`?

Reply:
Nah I don't think there's any reason not to just call it `fftw.py`